### PR TITLE
Breaking Change: Updating async methods to follow C# async naming conventions

### PR DESCRIPTION
**Pull request by hpsin**
And updating version number.  This handles #183

### CODE REVIEW

File: WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/Dns-Sd.cs [hunk 1]
```diff
@@ -41,30 +41,30 @@ public partial class DevicePortal
...
         {
-            await this.Post(
+            await this.PostAsync(
                 TagApi,
                 string.Format("tagValue={0}", tagValue));
         }
 
         /// <summary>
-        /// Delete all tags from the device's DNS-SD broadcast. 
+        /// DeleteAsync all tags from the device's DNS-SD broadcast.
```
Comment (MattHyman) [inline WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/Dns-Sd.cs]: Should this just be "Delete"?

File: WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/Dns-Sd.cs [hunk 2]
```diff
@@ -41,30 +41,30 @@ public partial class DevicePortal
...
-        public async Task DeleteAllTags()
+        public async Task DeleteAllTagsAsync()
         {
-            await this.Delete(TagsApi);
+            await this.DeleteAsync(TagsApi);
         }
 
         /// <summary>
-        /// Delete a specific tag from the device's DNS-SD broadcast. 
+        /// DeleteAsync a specific tag from the device's DNS-SD broadcast.
```
Comment (MattHyman) [inline WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/Dns-Sd.cs]: Should this also just be "Delete"?

File: WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Xbox/UserManagement.cs [hunk 3]
```diff
@@ -142,7 +142,7 @@ public class UserInfo
             /// <summary>
             /// Gets or sets if the user should be deleted
             /// </summary>
-            [DataMember(Name = "Delete", EmitDefaultValue = false)]
+            [DataMember(Name = "DeleteAsync", EmitDefaultValue = false)]
```
Comment (MattHyman) [inline WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Xbox/UserManagement.cs]: Just "Delete" here.

File: WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.UniversalWindows/HttpRest/RestDelete.cs [hunk 4]
```diff
@@ -18,7 +18,7 @@
 namespace Microsoft.Tools.WindowsDevicePortal
 {
     /// <content>
-    /// Universal Windows Platform implementation of HTTP Delete
+    /// Universal Windows Platform implementation of HTTP DeleteAsync
```
Comment (MattHyman) [inline WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.UniversalWindows/HttpRest/RestDelete.cs]: Just "Delete" here.

**Comment by hpsin** [comment]
This set I purposefully kept as *Async, since it’s the implementation of a required method.

**Comment by MattHyman** [comment]
Hirsch,

I actually deleted that comment shortly after creating it because I caught what you discussed below. Sorry for the noise.

Thanks,

Matt

**Pull request merged**